app/views/research_sessions/preview.html.erb

```diff
@@ -269,11 +266,17 @@
       participation is voluntary and that I am free to withdraw them at any time
       without giving any reason.
     </li>
-    <li>I understand that my <%='childʼs' if @research_session.unable_to_consent? %>
-      activities during the research session may be observed and will be recorded.
-      The data captured, in the form of <%= @research_session.recording_methods_sentence %>,
-      will be used for current and future service development.
-    </li>
+    <%=
+      react_component(
+        'Shared/WhichWeWillRecordUsingSentence',
+        preview_params(:recording).merge(
+          all_recording_methods: RecordingMethods::NAME_VALUES,
```

Comment:
Probably should become part of tested `component_params` (later, needs a card), @rgarner

Reply:
(we need to find a better way of passing `all_recording_methods` in so that we don't repeat ourselves here and in the live preview)